target/linux/realtek/image/rtl930x.mk

```diff
@@ -1,5 +1,18 @@
 # SPDX-License-Identifier: GPL-2.0-only
 
+define Device/zyxel_xgs1210-12
+  SOC := rtl9302
+  UIMAGE_MAGIC := 0x93001210
+  ZYXEL_VERS := ABTY
+  DEVICE_VENDOR := Zyxel
+  DEVICE_MODEL := XGS1210-12
+    IMAGE_SIZE := 13312k
```

Comment:
indentation seems to be off here; not sure what's going on though; normally these are 1 tab indented; but here it's 2 spaces, except for this one.

Reply:
Oops. Will be fixed